src/pyshark/packet/common.py

```diff
@@ -24,11 +30,12 @@ def __setstate__(self, data):
             setattr(self, key, val)
 
 
-class StrWriter:
-    """A class which mocks the py.io.TerminalWriter to write to an internal buffer"""
-
-    def __init__(self):
-        self.buffer = ""
-
-    def write(self, text, *_, **__):
-        self.buffer += text
+@functools.wraps(termcolor.colored)
+def colored(text, *args, **kwargs):
+    try:
+        enable_color = sys.stdout.isatty()
+    except:
```

Comment:
Please don't use blank `except:`. Not sure which exception could be raised here but catch is specifically

Reply:
Normally I'd agree but that particular method is quite a murky one. `isatty()` may not be defined or `sys.stdout` is None (both leading to an `AttributeError()`), or raise a `NotImplementedError()` or even a `FileNotFoundError()`.

Reply:
Alright, I've put in the exceptions I know it can raise.